src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Rectangle implements Figure {
+    private String color;
+    private int length;
+    private int width;
+
+    public Rectangle() {
+        this.length = length;
+        this.width = width;
+        this.color = color;
```

Comment:
The constructor is trying to assign values to `length`, `width`, and `color` but these parameters are not provided in the constructor's parameter list. To fix this, the constructor should take three parameters: `Rectangle(int length, int width, String color)` and use these to initialize the fields.